Makefile

```diff
@@ -65,7 +65,7 @@ $(PYTHON):
 build-requirements:
 	$(VIRTUALENV) $(TEMPDIR)
 	$(TEMPDIR)/bin/pip install -U pip
-	$(TEMPDIR)/bin/pip install -Ue ".[monitoring,postgresql,memcached]"
+	$(TEMPDIR)/bin/pip install -Ue ".[monitoring,postgresql,memcached,mohawk]"
```

Comment:
nitpick: I think it could be just `hawk` since the underlying library may change overtime :)

Reply:
good call!